scripts/limonade/src/index.ts

```diff
@@ -180,6 +187,16 @@ async function run() {
       filters,
     }
   );
+  connection.onSlotChange(() => {
+    if (solanaConnectionTimeout !== undefined) {
+      clearTimeout(solanaConnectionTimeout);
+    }
+
+    solanaConnectionTimeout = setTimeout(() => {
+      throw new Error("Solana websocket timeout");
```

Comment:
maybe don't crash, just create a log for alerts? idk what the best practice is, but if it's just a momentary issue, maybe it's better to keep it alive? maybe this is bad hygeine

Reply:
yeah I'm on kind of on the fence too

Reply:
I think it's good because on restart it will into `submitExistingOpportunities` with the updates that we missed for 30 seconds